config/service-brokers/aiven/config.json

```diff
@@ -15,12 +15,35 @@
           "supportUrl": "https://www.cloud.service.gov.uk/support"
         },
         "plans": [
+          {
+            "id": "07264114-c666-440b-934a-015508be4475",
+            "name": "tiny-5.x",
+            "aiven_plan": "startup-4",
+            "elasticsearch_version": "5",
+            "description": "NON-HA, 1 dedicated VM, 1 CPU per VM, 4GB RAM per VM, 80GB disk space.",
+            "free": true,
+            "metadata": {
+              "displayName": "Tiny, NON-HA single-node Elasticsearch 5 cluster"
+            }
+          },
+          {
+            "id": "7c0e6f6a-e443-41a0-83df-981bd35923a9",
+            "name": "tiny-6.x",
+            "aiven_plan": "startup-4",
+            "elasticsearch_version": "6",
+            "description": "NON-HA, 1 dedicated VM, 1 CPU per VM, 4GB RAM per VM, 80GB disk space.",
+            "free": true,
+            "metadata": {
+              "displayName": "Tiny, NON-HA single-node Elasticsearch 6 cluster"
+            }
+          },
           {
             "id": "0eee9e10-db37-4cc3-bc9b-39e378ff3a5f",
             "name": "small-ha-5.x",
             "aiven_plan": "business-4",
             "elasticsearch_version": "5",
             "description": "3 dedicated VMs, 1 CPU per VM, 4GB RAM per VM, 240GB disk space.",
+            "free": false,
```

Comment:
On a slightly unrelated note, does our billing app take this field into account?

Reply:
It doesn't, and it shouldn't really. In our context, "free" means can be used by people in a trial. Non-trial users will still be expected to pay for it though.

In other news, naming things is hard.